toy_data/improved/create/include_required.md

```diff
@@ -0,0 +1,81 @@
+# Required elements of the INCLUDE model
+
+We need to identify required elements of the [INCLUDE model](https://github.com/include-dcc/include-linkml).
+
+Looking through the separate YAML files, there are no actionable elements in the include_core.yaml and include_schema.yaml files. These files will be handled during data conversion as they are part of the the schema is organized rather than describing core fields that need to contain data.
+
+Here are the remaining data files and their required classes with the classes required slots.
+
+ - include_study.yaml
+    - Study
+        - studyCode
+        - studyTitle
+        - program
+        - studyDescription
+        - studyContactName
+        - studyContactInstitution
+        - studyContactEmail
+        - researchDomain
+        - participantLifespanStage
+        - studyDesign
+        - clinicalDataSourceType
+        - guidType
+    - Dataset
+        - studyCode
+        - datasetName
+        - datasetGlobalId
+        - expectedNumberOfFiles
+        - dataCollectionStartYear
+        - dataCollectionEndYear
+        - dataCategory
+        - dataType
+        - experimentalStrategy
+        - experimentalPlatform
+        - accessLimitations
+        - accessRequirements
+ - include_participant.yaml
+    - Participant
+        - studyCode
+        - participantGlobalId
+        - participantExternalId
+        - familyType
+        - familyRelationship
+        - sex
+        - race
+        - ethnicity
+        - downSyndromeStatus
+        - ageAtFirstPatientEngagement
+        - firstPatientEngagementEvent
+- include_assay.yaml
+    - Biospecimen
+        - studyCode
+        - participantGlobalId
+        - participantExternalId    
+        - sampleGlobalId
+        - sampleExternalId
+        - sampleType
+        - sampleAvailability
+        - containerAvailability
+    - Datafile
+        - studyCode
+        - fileName
+        - fileGlobalId
+        - fileS3Location
+        - drsUri
+        - dataAccess
+        - dataCategory
+        - dataType
+        - fileFormat
+
+
+ ## Notes
+ Here are some notes from the evaluation of the INCLUDE data model
+
+ ### Dataset
+ The slots datasetGlobalID, expectedNumberOfFiles, dataCollectionStartYear, accessLimitations, and accessRequirements are marked required=false with a note to change to true when something is figured out; I'm going to consider these true for simplicity.
```

Comment:
I think these should still be considered `required=false` here since I don't think there has been a decision to require these or even if or how to use these, however Madan as maintainer of the model will know better. 

@madanucd has there been any further discussion you are aware of in INCLUDE that will make these fields required?

Reply:
@twhetzel and @madanucd I haven't updated any of this to reflect the branch you showed me in the INCLUDE model repository. When you get those merged in we can update this. Alternatively, if either of you would like to make whatever changes you think need to be made please do!

Reply:
I'm not sure what branch you are referring to. Madan is the maintainer of the INCLUDE model.